### PR TITLE
Minor ui tweaks

### PR DESCRIPTION
**Pull request by HSalmon**

### CODE REVIEW

File: config/locales/client.en.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ en:
 
   start_proposal_card:
     title: 'Start a proposal'
-    helptext: Ready to make a decision? Proposals let you see how everyone feels about a particular course of action.
+    helptext: Ready to make a decision?
```
Comment (robguthrie) [inline config/locales/client.en.yml]: I really like the "See how people feel about a course of action" part.. I wonder if there is room for that bit, @rdbartlett

**Comment by robguthrie** [comment]
Can you squash then I'm happy to merge.

File: config/locales/client.en.yml (hunk 1 above)
Comment (rdbartlett) [inline config/locales/client.en.yml]: That text is duplicated on the start proposal modal so I don't think we are losing anything.

**Comment by HSalmon** [comment]
@robguthrie - done

**Comment by robguthrie** [comment]
:ear: ChEARS

**Pull request merged**